packages/eslint-plugin-tslint/package.json

```diff
@@ -28,6 +28,7 @@
     "lodash.memoize": "^4.1.2"
   },
   "peerDependencies": {
+    "eslint": ">= 5 < 6",
```

Comment:
We could probably just use `^5.0.0`.

Reply:
Fixed in [2d2826f](https://github.com/typescript-eslint/typescript-eslint/pull/151/commits/2d2826f1faa01230122ec2e6c7df4bef9f28d99f).